WalletWasabi.Tests/UnitTests/EventSourcing/InMemoryEventRepositoryTests.cs

```diff
@@ -0,0 +1,433 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WalletWasabi.EventSourcing;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Interfaces.EventSourcing;
+using WalletWasabi.Tests.UnitTests.EventSourcing.TestDomain;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace WalletWasabi.Tests.UnitTests.EventSourcing
+{
+	public class InMemoryEventRepositoryTests
+	{
+		private const string TestRoundAggregate = "TestRoundAggregate";
+		private readonly TimeSpan _semaphoreWaitTimeout = TimeSpan.FromSeconds(1);
```

Comment:
This is unused at the moment.

Reply:
It is:
Code	File	Line	Column	Project	Containing Member	Containing Type	Kind	Repository
Assert.True(TestEventRepository.AppendedSemaphore.Wait(_semaphoreWaitTimeout));	C:\Users\Lajos\git\WalletWasabi\WalletWasabi.Tests\UnitTests\EventSourcing\InMemoryEventRepositoryTests.cs	234	60	WalletWasabi.Tests	AppendEvents_CriticalSectionConflicts_Async	InMemoryEventRepositoryTests	Read	
Assert.True(TestEventRepository.ConflictedSemaphore.Wait(_semaphoreWaitTimeout));	C:\Users\Lajos\git\WalletWasabi\WalletWasabi.Tests\UnitTests\EventSourcing\InMemoryEventRepositoryTests.cs	245	62	WalletWasabi.Tests	AppendEvents_CriticalSectionConflicts_Async	InMemoryEventRepositoryTests	Read	
Assert.True(TestEventRepository.AppendedSemaphore.Wait(_semaphoreWaitTimeout));	C:\Users\Lajos\git\WalletWasabi\WalletWasabi.Tests\UnitTests\EventSourcing\InMemoryEventRepositoryTests.cs	275	60	WalletWasabi.Tests	AppendEvents_CriticalAppendConflicts_Async	InMemoryEventRepositoryTests	Read	
Assert.False(TestEventRepository.ConflictedSemaphore.Wait(_semaphoreWaitTimeout));	C:\Users\Lajos\git\WalletWasabi\WalletWasabi.Tests\UnitTests\EventSourcing\InMemoryEventRepositoryTests.cs	286	63	WalletWasabi.Tests	AppendEvents_CriticalAppendConflicts_Async	InMemoryEventRepositoryTests	Read

Reply:
Ah, release configuration again. Tx.